src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Product.xml

```diff
@@ -18,7 +18,8 @@
     <resource class="%sylius.model.product.class%" shortName="Product">
         <attribute name="normalization_context">
             <attribute name="groups">
-                <attribute>product:read</attribute>
+                <attribute>admin:product:read</attribute>
```

Comment:
If you didn't change it in all of the other places, let's leave admin serialization without the prefix
```suggestion
                <attribute>product:read</attribute>
```

Reply:
We should introduce this prefix in separate PR